OMCompiler/SimulationRuntime/OMSICpp/CMakeLists.txt

```diff
@@ -921,15 +917,15 @@ IF(ZeroMQ_FOUND AND cppzmq_FOUND)
     IF((MSVC) AND (CMAKE_BUILD_TYPE MATCHES Debug))# workarround, find_package(ZeroMQ)  fails because wrong filename under windows and VS projects
       get_target_property(ZeroMQ_LIB libzmq  IMPORTED_IMPLIB_DEBUG)
        message (STATUS "Using workarround for ${ZeroMQ_LIB} " )
-    ELSE()
+    ELSE((MSVC) AND (CMAKE_BUILD_TYPE MATCHES Debug))
       set(ZeroMQ_LIB  ${ZeroMQ_LIBRARY})
-    ENDIF(MSVC)
+    ENDIF((MSVC) AND (CMAKE_BUILD_TYPE MATCHES Debug))
 
-    message (STATUS "Using ZMQ library ${ZeroMQ_LIBRARY} ${ZeroMQ_LIB} " )
-    message(STATUS "cppzmq version ${cppzmq_VERSION}")
-    message (STATUS "Using ZMQ Header ${ZeroMQ_INCLUDE_DIR}" )
-    message (STATUS "Using CPPZMQ Header ${cppzmq_INCLUDE_DIR}" )
-ELSE(ZeroMQ_FOUND AND cppzmq_FOUND)
+    message (STATUS "Using ZMQ library ${ZeroMQ_LIBRARY} ${ZeroMQ_LIB} " )
+    message (STATUS "cppzmq version ${cppzmq_VERSION}")
+    message (STATUS "Using ZMQ Header ${ZeroMQ_INCLUDE_DIR}" )
+    message (STATUS "Using CPPZMQ Header ${cppzmq_INCLUDE_DIR}" )
+ELSE(ZeroMQ_found AND cppzmq_found)
```

Comment:
Why are these lowercase. CMake won't care but if it signifies that a package is found it is usually _FOUND.